### ISSUE TITLE
extras migration 21 missing dependencies

### ISSUE THREAD

**Issue by gsnider2195** [nautobot/nautobot#4657]
Missing dependencies for models from `admin` and `contenttypes`

```
  Applying extras.0021_customfield_changelog_data...Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/django/apps/registry.py", line 156, in get_app_config
    return self.app_configs[app_label]
           ~~~~~~~~~~~~~~~~^^^^^^^^^^^
KeyError: 'admin'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/nautobot-server", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/usr/local/lib/python3.11/site-packages/nautobot/core/cli/__init__.py", line 54, in main
    run_app(
  File "/usr/local/lib/python3.11/site-packages/nautobot/core/runner/runner.py", line 297, in run_app
    management.execute_from_command_line([runner_name, command] + command_args)
  File "/usr/local/lib/python3.11/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.11/site-packages/django/core/management/__init__.py", line 413, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.11/site-packages/django/core/management/base.py", line 354, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.11/site-packages/django/core/management/base.py", line 398, in execute
    output = self.handle(*args, **options)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/core/management/base.py", line 89, in wrapped
    res = handle_func(*args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/core/management/commands/migrate.py", line 244, in handle
    post_migrate_state = executor.migrate(
                         ^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/migrations/executor.py", line 117, in migrate
    state = self._migrate_all_forwards(state, plan, full_plan, fake=fake, fake_initial=fake_initial)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/migrations/executor.py", line 147, in _migrate_all_forwards
    state = self.apply_migration(state, migration, fake=fake, fake_initial=fake_initial)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/migrations/executor.py", line 227, in apply_migration
    state = migration.apply(state, schema_editor)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/migrations/migration.py", line 126, in apply
    operation.database_forwards(self.app_label, schema_editor, old_state, project_state)
  File "/usr/local/lib/python3.11/site-packages/django/db/migrations/operations/special.py", line 190, in database_forwards
    self.code(from_state.apps, schema_editor)
  File "/usr/local/lib/python3.11/site-packages/nautobot/extras/migrations/0021_customfield_changelog_data.py", line 61, in migrate_history
    LogEntry = apps.get_model("admin", "LogEntry")
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/apps/registry.py", line 206, in get_model
    app_config = self.get_app_config(app_label)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/apps/registry.py", line 163, in get_app_config
    raise LookupError(message)
LookupError: No installed app with label 'admin'.
```

**Comment by gsnider2195** [comment]
Looks like there are some migrations with missing contenttypes dependencies as well:

- nautobot/circuits/migrations/0017_fixup_null_statuses.py
- nautobot/dcim/migrations/0034_migrate_region_and_site_data_to_locations.py
- nautobot/dcim/migrations/0046_fixup_null_statuses.py
- nautobot/extras/migrations/0021_customfield_changelog_data.py
- nautobot/extras/migrations/0024_job_data_migration.py
- nautobot/extras/migrations/0062_collect_roles_from_related_apps_roles.py
- nautobot/extras/migrations/0077_migrate_custom_field_data.py
- nautobot/ipam/migrations/0016_prefix_type_data_migration.py
- nautobot/ipam/migrations/0022_aggregate_to_prefix_data_migration.py
- nautobot/ipam/migrations/0031_ipam___data_migrations.py
- nautobot/ipam/migrations/0033_fixup_null_statuses.py
- nautobot/virtualization/migrations/0024_fixup_null_statuses.py

**Comment by glennmatthews** [comment]
Addressed in #5323.
